### PR TITLE
Sql script

### PR DESCRIPTION
**Pull request by paytonshaltis**
Rectifies concerns with database files in the repository.

- Adds `db.sqlite` to the `.gitignore` file.
- Adds `seed.js` which deletes, recreates, and reseeds a local instance of the database for development use.
  - Includes 2 sample accounts to be seeded into the database.
  - More data can be added here as we get further into development.
- Adds the npm script `npm run seed` to recreate a local instance of the database.

### CODE REVIEW

**Comment by sderacy** [comment]
Looks good, I changed the package.json a bit since I figured it would make sense to format the code each time to run the program. You should be able to just run it by itself but it's a little npm trick I learned that helps shorten how much work we have to do

**Comment by sderacy** [comment]
I don't wanna tell you how to code so the last thing I would say is to change the "Starting the Server" section to have an instruction that says to run "npm run seed" for each clean installation

File: README.md [hunk 1]
```diff
@@ -58,9 +58,22 @@ If you get a message about files being modified and changing from LF to CLF for
...
 - Because each user will likely have a different number of social media accounts, it makes more sense to store these in a seperate table and link the two with a foreign key.
+
+### Local Database Setup
+
+In order to run the app locally, the database also needs to exist in the project directory. In order to prevent the database from being committed to the repo, it is ignored by git. To create the database, run `npm run seed` in the main project directory. This will:
+
+1. Delete the database file if it already exists.
+2. Create a new database file.
+3. Create the tables.
+4. Seed the database with some sample data from `./seed.js`.
```
Comment (sderacy) [inline README.md:77]: Actually scratch my most recent comment.

**Review by sderacy — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (paytonshaltis) [reply README.md:77]: > Actually scratch my most recent comment.

Haha, I was just about to mention this. Np.

**Pull request merged**